proof/refine/RISCV64/Invariants_H.thy

```diff
@@ -7,9 +7,15 @@
 theory Invariants_H
 imports
   LevityCatch
-  "AInvs.ArchDetSchedSchedule_AI"
+  "AInvs.AInvs"
+  "Lib.AddUpdSimps"
+  ArchMove_R
 begin
 
+(* FIXME: this should go somewhere in spec *)
+translations
+  (type) "'a kernel" <=(type) "kernel_state \<Rightarrow> ('a \<times> kernel_state) set \<times> bool"
```

Comment:
Indeed it should :-)

Reply:
This translation was already in `KernelStateData_H.thy`, so I've removed this